appveyor.yml

```diff
@@ -0,0 +1,28 @@
+branches:
+  only:
+    - master
+init:
+  - git config --global core.autocrlf input
+environment:
+  matrix:
+    - nodejs_version: '8'
+      webpack_version: 2.6.0
+      job_part: test
+    - nodejs_version: '6'
+      webpack_version: 2.6.0
+      job_part: test
+    - nodejs_version: '4.3'
+      webpack_version: 2.6.0
+      job_part: test
+build: 'off'
+matrix:
+  fast_finish: true
+install:
+  - 'ps: Install-Product node $env:nodejs_version x64'
```

Comment:
You have to strip the `'` off of anything that has `ps: || cmd:`. Already has an open issue in defaults.